sample/src/main/res/values/ids.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+</resources>
```

Comment:
What's this?

Reply:
Ah, that's are leftovers from my experiments. Will remove it.